src/sail_lean_backend/Sail/Sail.lean

```diff
@@ -15,5 +15,16 @@ def truncate {w: Nat} (x: BitVec w) (w': Nat) : BitVec w' :=
 def truncateLSB {w: Nat} (x: BitVec w) (w': Nat) : BitVec w' :=
   x.extractLsb' 0 w'
 
+def extractLsb {w: Nat} (x: BitVec w) (hi lo: Nat) : BitVec (hi - lo + 1) :=
+  x.extractLsb hi lo
```

Comment:
```suggestion
def extractLsb {w : Nat} (x : BitVec w) (hi lo : Nat) : BitVec (hi - lo + 1) :=
  x.extractLsb hi lo
```

Lean style puts a space before the `:` in arguments. I did notice the rest of the file has the same formatting, so we might want to address this in a separate formatting PR and fix the whole file in one go.